### DISCUSSION TITLE
unable to use VHTML in vite

### DISCUSSION THREAD

**Discussion by leamandeep** [preactjs/wmr#961] [Q&A]
I tried to configure vhtml in vite's vanilla javascript template but I'm unable to implement it. Please guide me on how to configure it. I will then implement vhtml in preact

```js
main.js
------

/** @jsx vhtml */
import { Counter } from "./src/counter";

document.body.innerHTML = (
  <main>
   <Counter/>
  </main>
);
```

```js
/** @jsx vhtml */

export function Counter() {
  let count = 0;
  return (
    <section>
      <h1>Click button</h1>
      <button onClick={() => count++}>Click me</button>
    </section>
  );
}
```

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Vite App</title>
  </head>
  <body>
    <div id="app"></div>
    <script type="module" src="./main.jsx"></script>
  </body>
</html>
```

![image](https://github.com/developit/vhtml/assets/68345524/110486f7-7fc5-4eb3-866e-bfdefbe9a929)

**Comment by rschristian** [comment]
Er, I think you might be a bit lost? This doesn't look to be related to WMR. Did you mean to post this on Preact itself?

Regardless, you'll need to provide more details, as I'm not quite sure what "vhtml" is supposed to be?
